cursorless-talon/src/spoken_forms.json

```diff
@@ -17,6 +17,7 @@
       "decrement": "decrement",
       "dedent": "outdentLine",
       "define": "revealDefinition",
+      "split define": "revealDefinitionAside",
```

Comment:
I'd be tempted to swap this: `"define split"`

Reply:
Totally agree

Reply:
Should it be "follow split"?

Reply:
Yeah